docs/book/beyond_basics/using_client_go_informers.md

```diff
@@ -0,0 +1,68 @@
+
+# Using Generated Informers
+
+This chapter describes how to use the client-go generated Informers with controller-runtime Watches.
+
+[Link to reference documentation](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/source#Informer)
+
+**Note:** The `source.Informers` and `source.Kind` sources create different caches.  Using both
+`source.Informers` and `source.Kind` sources in the same project will result in duplicate caches.
+
+{% method %}
+## Creating client-go generated Informers and Adding them to the Manager
+
+Instantiate the generated InformerFactory and add it to the Manager so it is started automatically.
+
+{% sample lang="go" %}
+```go
+// Create the InformerFactory
+generatedClient := kubernetes.NewForConfigOrDie(mgr.GetConfig())
+generatedInformers := kubeinformers.NewSharedInformerFactory(generatedClient, time.Minute*30)
+
+err := mgr.Add(manager.RunnableFunc(func(s <-chan struct{}) error {
+    generatedInformers.Start(s)
+    return nil
+}))
```

Comment:
I haven't seen may people using the `RunnableFunc` adaptor to create a controller, so may be having an example that demonstrates it with controller will be better ?